src/transformers/models/bert/modeling_bert.py

```diff
@@ -1128,7 +1127,7 @@ def forward(
     """Bert Model with a `language modeling` head on top for CLM fine-tuning.""", BERT_START_DOCSTRING
 )
 class BertLMHeadModel(BertPreTrainedModel):
-    _tied_weights_keys = ["predictions.decoder.bias", "cls.predictions.decoder.weight"]
+    _tied_weights_keys = ["cls.predictions.decoder.bias", "cls.predictions.decoder.weight"]
```

Comment:
Seems like this was a bug,  `predictions` does not exist onthis model, only  `cls.predictions`.